index.js

```diff
@@ -4,6 +4,8 @@
  * MIT Licensed
  */
 
+import 'core-js/stable'
+import 'regenerator-runtime/runtime'
```

Comment:
I don't want to have to import these polyfills. We should instead look to do simple feature detection. If Symbols are not available in the runtime we can simply not print them.

Reply:
Ok, I'll try to do something with Symbols. It would be great if you took the time to implement this detection, since I'm not very experienced

Reply:
We can do some simple feature detection, such as `if typeof Symbol !== 'undefined`, or a more advanced check could be `if typeof Symbol === 'function' && typeof Symbol() === 'symbol'`. I would suggest wrapping both tests and logic in these checks, and writing additional tests that can assert when Symbols aren't available.

Reply:
I've added some code to skip the failing tests, we might now just want to add more for the inverse condition - to make a test that passes when Symbols aren't available, asserting on the behavior.